backend/settings/base.py

```diff
@@ -40,8 +40,16 @@
     'app',
     'cloudinary',
     'rest_framework',
+    'rest_framework.authtoken',
 ]
 
+REST_FRAMEWORK = {
+    'DEFAULT_AUTHENTICATION_CLASSES': [
+        'rest_framework.authentication.TokenAuthentication',
+    ],
+}
+
+
 MIDDLEWARE = [
     'django.middleware.security.SecurityMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
```

Comment:
Notice that the Travis build failed? If you read the build log, you'll see that the error is a 401 unauthorized error. The test users also need to have some test authentication so it also needs to implemented in the test files too. Maybe clone [Delores project](https://github.com/Del-sama/Django_ams_api) and use it as a guide. I'll see if I can find articles that cover this topic. Also, here's a [list](https://www.restapitutorial.com/httpstatuscodes.html) of error codes and their meanings. Overall, you're on the right track.